Content.Server/Temperature/Components/TemperatureComponent.cs

```diff
@@ -22,22 +24,28 @@ public class TemperatureComponent : Component
         /// <inheritdoc />
         public override string Name => "Temperature";
 
-        [DataField("coldDamageType",required: true)]
-        private readonly string coldDamageType = default!;
-        [DataField("hotDamageType",required: true)]
-        private readonly string hotDamageType = default!;
+        // TODO QUESTION Similar to the asphyxiation question, does this actually need to be a datafield?
```

Comment:
i'd keep them as is, doesn't hurt and most stuff will just use the defaults

Reply:
I also realise that if this were fixed, then it wouldn't be possible for someone to change the damage types by editing yaml files. This way they can, though they then need to override defaults to avoid errors.